src/coverlet.core/coverlet.core.csproj

```diff
@@ -1,14 +1,16 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <OutputType>Library</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
Why do you need to change this to net oreapp now?

Reply:
I needed to be able to use `Path.GetRelativePath`. It doesn't really matter much though since consumers of coverlet do not explicitly depend on this assembly